pytest_harvest/results_session.py

```diff
@@ -443,6 +446,8 @@ def get_pytest_params(item):
     if isinstance(item, _MinimalItem):
         # Our special _MinimalItem object - when xdist is used and worker states have been saved + restored
         return item.get_pytest_params()
+    elif isinstance(item, DoctestItem):
+        return OrderedDict()
```

Comment:
Same that earlier: can you please insert a comment explaining why this can only be an empty dict ? I guess that this is because doctests can not receive fixtures or parameters, but I did not check that this was true